src/actors/redis-generator/redis-generator.py

```diff
@@ -0,0 +1,89 @@
+import os
+import sys
+import json
+import shlex
+import subprocess
+from distutils.spawn import find_executable
+
+
+def find_buildah():
+    def _is_valid(path):
+        return path and os.path.isfile(path) and os.access(path, os.X_OK)
+    path = os.path.abspath('buildah')
+    if not _is_valid(path):
+        path = find_executable('buildah')
+    if not _is_valid(path):
+        raise Exception('Could not find a valid buildah binary: {0}'.format(path))
+    return path
+
+
+BUILDAH = find_buildah()
+
+
+def _execute(cmd):
+    out = subprocess.check_output(shlex.split(cmd))
+    return out.strip()
+
+
+def _create_redis_container(redis):
+    # TODO: use a RHEL/CentOS-based image
+    cname = _execute('{buildah} from redis:{version}-alpine'.format(
+        buildah=BUILDAH,
+        version=redis['version'])
+    )
+    if not cname:
+        raise Exception('Error creating container')
+    return cname
+
+
+def _copy_to_container(cname, src, dst):
+    if cname and src and dst:
+        _execute('{buildah} copy {cname} {src} {dst}'.format(
+            buildah=BUILDAH,
+            cname=cname,
+            src=src,
+            dst=dst
+        ))
+
+
+def _get_container_id(cname):
+    cntrs = json.loads(_execute('{buildah} containers --json'.format(buildah=BUILDAH)))
+    ids = [x['id'].strip() for x in cntrs if x['containername'] == cname]
+    if len(ids) != 1:
```

Comment:
Do you have real case for that? I thinks it's impossible to have containers with same names

Reply:
As we talked personally, there shouldn't be two containers with the same name; however, we decided to keep this check just in case.

Reply:
My bad, it's better to check this as we discussed